site/app/templates/grading/electronic/Details.twig

```diff
@@ -470,19 +470,19 @@
             {% set contents = "Grade" %}
             {% set btn_class = "btn-primary" %}
 
-        {% if graded_gradeable.getOrCreateTaGradedGradeable().anyGrades() %}
+        {% if graded_gradeable.getOrCreateTaGradedGradeable().anyGrades(core.getUser()) %}
```

Comment:
Using `core` in Twig files is discouraged.  Instead, you can pass the current user into the template.

Reply:
@williamjallen Perhaps an issue for cleaning out usage of core in Twig should be made? There are many such cases in the code, I could see how an external contributor would think it is good practice